addon/services/asset-loader.js

```diff
@@ -80,7 +80,7 @@ export default Ember.Service.extend({
     if (cachedPromise) {
       return cachedPromise;
     }
-
+    
```

Comment:
can you remove the extraneous whitespace here?

Reply:
done